.github/workflows/static-analysis.yml

```diff
@@ -0,0 +1,33 @@
+name: Static analysis
+on:
+  repository_dispatch:
+  workflow_dispatch:
+  push:
+    branches: [9.x]
```

Comment:
it should run on all the 9.*.x branches - + on every PR if possible. else we could just say every branch as well.